.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+<!-- *Delete this comment before submitting*
```

Comment:
<!-- Test comment
This is a test
-->

Why does this need to be deleted, I thought HTML comments don't show up in issues anyway?

Reply:
Yes, it actually makes no sense. My intention was, that the users don't write their message content into the comment by accident.

Reply:
Closing the second comment at the end should help to avoid that :)